paddle/phi/ops/yaml/legacy/static_ops.yaml

```diff
@@ -993,6 +993,15 @@
   backward : unsqueeze_grad
   interfaces : paddle::dialect::InferSymbolicShapeInterface
 
+- op: concat_tensor
```

Comment:
move to paddle/phi/ops/yaml/inconsistent/static_ops.yaml

Reply:
done